redis/utils.py

```diff
@@ -117,3 +118,12 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+logger = logging.getLogger("push_response")
```

Comment:
instead, can this be a function? So the first time you'd call it, meaning the logger is triggered. Something like

def setup_logger(logevel=logging.INFO, handler=[logging.StreamHandler()]:
# do things here

This way, a user can also use our log handler and instantly override, just override their own handler if they want.

We need to only export the **name** of the logger. Then after the first time getLogger is called by python.. it's generally available